containers/init/ready/ready.go

```diff
@@ -269,4 +278,22 @@ func main() {
 	log.Printf("all pods ready, exiting successfully")
 	workerFileBody := strings.Join(podIPs, ",")
 	ioutil.WriteFile(outputFile, []byte(workerFileBody), 0777)
+
+	test, err := grpcClientset.LoadTestV1().LoadTests(corev1.NamespaceDefault).Get(ctx, os.Args[1], metav1.GetOptions{})
+	if err != nil {
+		log.Fatalf("failed to fetch loadtest for obtaining metadata: %v", err)
+	}
+
+	outputMetadataFile := DefaultMetadataOutputFile
+	outputMetadataFileOverride, ok := os.LookupEnv(OutputMetadataEnv)
+	if ok {
+		outputMetadataFile = outputMetadataFileOverride
+	}
+
+	metaDataSet := test.ObjectMeta
```

Comment:
Is this the right information, i.e., only the "metadata"  field?

Reply:
The information typically included are:
```
{
    "annotations": {
        "buildUrl": "wanlin-test",
        "gitActualCommit": "wanlin-test",
        "gitCommit": "wanlin-test",
        "jobName": "wanlin-test",
        "scenario": "go_generic_sync_streaming_ping_pong_secure",
        "uniquifier": "basic"
    },
    "creationTimestamp": "2021-08-10T22:06:37Z",
    "generation": 2,
    "labels": {
        "language": "go",
        "prefix": "examples"
    },
    "name": "examples-go-generic-sync-streaming-ping-pong-secure-basic",
    "namespace": "default",
    "resourceVersion": "139781494",
    "uid": "77870d26-8382-495e-b311-64b5e64dce40"
}
```
I ignored the annotations attached by kubectl, which only exit when we apply the LoadTest configuration through command line.
I have also omit the managedFields, which are not metadata.